frontend/test/views/AccountView.spec.js

```diff
@@ -75,4 +85,52 @@ describe('AccountView.vue', () => {
       expect(wrapper.vm.$router.push.called).to.be.true;
     });
   });
+
+  it('should toggle the secret value on click', async () => {
+    const wrapper = getMountedComponent();
+
+    let sectionBox = wrapper.findComponent('[data-test=credentials]');
+
+    await sectionBox.trigger('click');
+    await wrapper.vm.$nextTick();
+
+    sectionBox = wrapper.findComponent('[data-test=credentials]');
+    expect(sectionBox.props('content')[1].value).toEqual([wrapper.vm.user.clientSecret]);
+  });
+
+  it('should not toggle the secret value on click again after it has been toggled', async () => {
+    const wrapper = getMountedComponent();
+
+    let sectionBox = wrapper.findComponent('[data-test=credentials]');
+
+    await sectionBox.trigger('click');
+    await wrapper.vm.$nextTick();
+
+    sectionBox = wrapper.findComponent('[data-test=credentials]');
+    expect(sectionBox.props('content')[1].value).toEqual([wrapper.vm.user.clientSecret]);
+    await sectionBox.trigger('click');
+    await wrapper.vm.$nextTick();
+
+    sectionBox = wrapper.findComponent('[data-test=credentials]');
+    expect(sectionBox.props('content')[1].value).toEqual([wrapper.vm.user.clientSecret]);
+  });
+
+  // Test case where clientSecret is not set
```

Comment:
The comment from https://github.com/uab-cgds-worthey/rosalution/pull/46/files/ac1f146c9e5fbcc5769bb3799c0bac96b964204f#r1125001474 feedback is also relevant here.  This comment is not necessary since the 'it' statement on line 119 says the same thing.

Reply:
Whoops, good catch.